src/core/dfn-finder.js

```diff
@@ -31,18 +31,17 @@ const unlinkable = new Set(["maplike", "setlike", "stringifier"]);
  * @param {string} parent
  * @param {string} name
  * @param {Record<string, HTMLElement[]>} definitionMap
- * @param {HTMLElement} idlElem
+ * @param {boolean} suppressWarnings
  */
-export function findDfn(defn, parent, name, definitionMap, idlElem) {
+export function findDfn(defn, parent, name, definitionMap, suppressWarnings) {
```

Comment:
should avoid boolean trap (suppressWarnings)... maybe make it string or ` {suppressWarnings = false} = {}`